src/app/todo-page/new-item/new-item.component.ts

```diff
@@ -27,8 +26,8 @@ export class NewItemComponent implements OnInit {
   @Output() changeItem = new EventEmitter<Todo[]>();
   @Output() sortItem = new EventEmitter<Todo[]>();
 
-  constructor(private todoService: TodoItemService, private connectionService: ConnectionService) {
-    this.subscription = this.connectionService.todoData$.subscribe(item => this.editItem(item));
+  constructor(private connectionService: ConnectionService) {
+    this.checkData();
```

Comment:
```suggestion
    this.checkData();
```

Why you don't call this inside ngOnInit?

Reply:
Moved to ngOnInit()